waterlog/src/Pages/Home.js

```diff
@@ -10,7 +10,7 @@ class HomeComponent extends Component {
         <AlertsMap />
         <Wastage />
         <p className="home-text">Tank Levels</p>
-        <TankLevels />
+        {/*TODO <TankLevels />*/}
```

Comment:
I would rather just take this out. I mean if you wrote up a ticket for it, or a tester sees it, trust me, you'll be notified of it. This is just dead code to me. But it's your call

Reply:
I will remove it, since I am not the one implementing it